easybuild/easyconfigs/i/iomkl/iomkl-2017a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2017a'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolchain Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MKL & OpenMPI."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2017.1.132'
+gcc_maj_min = '6.3'
+gccver = '%s.0' % gcc_maj_min
```

Comment:
@akesandgren you can just drop `gcc_maj_min`, `gccver` and `binutilsver` not used anywhere

```python
gccsuff = '-GCC-6.3.0-2.27'
```